admin_manual/configuration_files/external_storage/local.rst

```diff
@@ -16,7 +16,7 @@ are on Ubuntu Linux::
  sudo -u www-data chmod -R 0750 /localdir
  
 See :ref:`label-setting-strong-perms` for information on finding your HTTP 
-user with ``phpinfo``. 
+user with :ref:`label-phpinfo`. 
  
```

Comment:
I suggest breaking this up into two parts, since it references two different pages: ' See :ref:`label-setting-strong-perms` for information on correct file permissions, and find your HTTP user with :ref:`phpinfo <label-phpinfo>` '